### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by MrGeorgen**
- command prompt update
- simplified code

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -87,16 +81,16 @@ if (config.antiAntiAFK) setInterval(function () {
...
+		userInput(cmd, false);
 		cmdInput();
 	});
 }
 
 // function to disconnect from the server
 function stop() {
 	webserver.isInQueue = false;
-	finishedQueue = false;
+	finishedQueue = config.minecraftserver.hostname !== "2b2t.org";
```
Comment (ChillerDragon) [inline main.js:93]: Maybe not hardcode this host everywhere one might want to slap another proxy in between.

Comment (MrGeorgen) [reply main.js:93]: Ok I'll add an config option

Comment (ChillerDragon) [reply main.js]: Awesome thanks!

Comment (MrGeorgen) [reply main.js]: No problem, it is nice to see that others read my code

**Review by themoonisacheese — APPROVED** [review]

**Pull request merged**